examples/tp-template/main.tf

```diff
@@ -43,12 +18,8 @@ module "trusted_profile_template" {
         key      = "serviceName"
         value    = "cloud-object-storage"
         operator = "stringEquals"
-        },
-        {
-          key      = "serviceInstance"
-          value    = module.cos.cos_instance_guid
-          operator = "stringEquals"
-      }]
```

Comment:
As discussed yesterday, the reason we added this was so we did not get policy clashes in our account as tests run. Each test would create a new COS instance meaning the policy attributes would always be unique. 
But since you mentioned this config doesn't really make sense lets try to change it in a way where it still wont cause policy clashes in our account. I think you mentioned a unique tag could be used maybe? var.prefix will always be unique in the tests, so if we could use that as part of a tag, it will always be unique